### PR TITLE
WC Issue: 26857 - Add action remove order item

### PR DESCRIPTION
**Pull request by mfs-mindsize**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

The purpose of this PR is to address the request of Issue #26857 "Add action for after remove_order_item"

Note: It also cleans up a stray line of code that my IDE recognized as not used. 

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### Changelog entry

> Dev - Added `woocommerce_ajax_order_items_removed` hook.

### CODE REVIEW

File: includes/class-wc-ajax.php [hunk 1]
```diff
@@ -1300,6 +1299,8 @@ public static function remove_order_item() {
 			$order->calculate_taxes( $calculate_tax_args );
 			$order->calculate_totals( false );
 
+			do_action( 'woocommerce_ajax_order_items_removed', $item_id, $item, $changed_stock, $order );
```
Comment (claudiosanches) [inline includes/class-wc-ajax.php:1302]: Please include a docblock for this hook. You can find instructions about it in https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters

**Review by claudiosanches — CHANGES_REQUESTED** [review]
Thanks for your help, it's just missing a docblock for this new hook.

File: includes/class-wc-ajax.php (hunk 1 above)
Comment (mfs-mindsize) [reply includes/class-wc-ajax.php:1312]: Added.

File: includes/class-wc-ajax.php [hunk 2]
```diff
@@ -1300,6 +1299,18 @@ public static function remove_order_item() {
 			$order->calculate_taxes( $calculate_tax_args );
 			$order->calculate_totals( false );
 
+			/**
+			 * Fires after order items are removed.
+			 *
+			 * @since 4.9.0
```
Comment (claudiosanches) [inline includes/class-wc-ajax.php:1305]:
```suggestion
			 * @since 5.1.0
```

**Review by claudiosanches — CHANGES_REQUESTED** [review]
Thanks for your help, there's only a small change required before we can merge this PR.
Thanks

File: includes/class-wc-ajax.php (hunk 2 above)
Comment (mfs-mindsize) [reply includes/class-wc-ajax.php]: @claudiosanches - Done :) If in the future I should not leave a msg, but instead should use "Resolve conversation" just let me know.

Comment (mfs-mindsize) [reply includes/class-wc-ajax.php]: @claudiosanches - Pardon my n00b-iness but can you tell me what I need to (un) do about this?

https://github.com/woocommerce/woocommerce/actions/runs/528661392

File: includes/class-wc-ajax.php [hunk 3]
```diff
@@ -1300,6 +1299,18 @@ public static function remove_order_item() {
 			$order->calculate_taxes( $calculate_tax_args );
 			$order->calculate_totals( false );
 
+			/**
+			 * Fires after order items are removed.
+			 *
+			 * @since 5.1.0
```
Comment (claudiosanches) [inline includes/class-wc-ajax.php:1305]:
```suggestion
			 * @since 5.2.0
```

**Review by claudiosanches — APPROVED** [review]
Thank you very much!

**Pull request merged**